CARTOGRAPHY.md

```diff
@@ -37,3 +37,67 @@ account.
+
+Apart from the characteristics of the data the most important criterion for deciding on rendering a certain type of feature or not is how visible and meaningful this feature is in reality.  Prominently visible elements tend to be important for orientation even if they are not particularly important themselves for a map user.  Some features like for example an ATM might not be very visible but serve an important social function in their surrounding and should therefore be shown.
+
+If several similar types of features should be rendered identically or differently and which features should differ strongly in appearance and which only in minor nuances should foremost be based on their difference in meaning and purpose for the target map users.  Differences in physical appearance are only relevant as far as they also represent differences in meaning and purpose.
+
+Using data that is not from the OSM database should be avoided - unless it is strictly necessary for the goals.  Preprocessing of OSM based data can be and is used (like for the coastline) but should carefully be considered regarding the influence this has on mapper feedback.
+
+Design of symbols, area and line patterns should be in a way that avoids their geometries to be mistaken as actual data.
+
+How something is rendered should - as far as possible within the guidelines and goals outlined so far - be in line with mappers' expectations.  If the way something is rendered appears either ugly, too prominent, contradicting or diametrical to readability in general mappers will try to avoid mapping it this way and if another feature class appears to look more according to the expectations its tagging is often used instead (also known as mapping for the renderer).  Similar things happen if the way some feature is rendered is very generic, not particularly indicative and distinct for the class of features it is meant to represent. It then will often be abused to map other types of features.  For these reasons it can be more productive to not render something than to render it in a suboptimal way.
+
+Also important in terms of mapper feedback is that how data influences the appearance of features in the map needs to be comprehensible for the viewer.  Complex relationships and unexpected interactions between different features should be avoided.
+
+One particular consideration due to this style being an important part of the public face of OpenStreetMap is to be careful and considerate with changes having a strong impact on the overall appearance of the map, especially with elements that have been rendered a certain way for a long time.  Such changes should be broadly discussed also beyond the issue tracker and comments and concerns should be carefully considered.
```

Comment:
I disagree here - Because of the map style being the face of OSM in many ways we need to be willing to discuss significant changes to this with the OSM community on a larger scale.  @matkoniecz did this quite well with the road color changes and if we should have other changes with similar impact in the future (boundary color and water color come to mind here) we should broaden the discussion on these as well.  This does not mean decisions should be made elsewhere but opinions should be considered even if they have only been articulated abroad.